etc/kayobe/kolla.yml

```diff
@@ -307,6 +307,16 @@ kolla_build_blocks:
     RUN sed -i 's/"pc-q35-rhel8.5.0"/"pc-q35-*"/' /usr/share/qemu/firmware/50-edk2-ovmf-cc.json
     {% endif %}
     {% endraw %}
+  neutron_server_footer: |
+    # This is a workaround for a bug in etcd3gw that was fixed in 1.0.1, but not in v0.2.3.
+    # https://opendev.org/openstack/etcd3gw/commit/ed899b34e464862525f76fff2377a2cceeb65af7
+    {% raw %}
+    {% if base_distro == 'centos' %}
+    RUN sed -i 's/decode_unicode=True/decode_unicode=False/' /var/lib/kolla/venv/lib/python3.6/site-packages/etcd3gw/watch.py
```

Comment:
You can use `python{{ distro_python_version }}`, but I'd forgive anyone for not knowing that, and not worth updating :)

Reply:
dammit, that would have been cleaner. Next time!